tests/syntax/test_pruning.py

```diff
@@ -176,68 +176,73 @@ def test_visibility_pruning():
         param p = foo.position
         """
     )
     positions = [sampleParamP(scenario, maxIterations=100) for i in range(30)]
-    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 1.1 for pos in positions)
+    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 2 for pos in positions)
     assert any(pos.distanceTo(Vector(0, 0, 0)) >= 1 for pos in positions)
 
     # visible
     scenario = compileScenic(
         """
         workspace = Workspace(RectangularRegion(0@0, 0, 1e10, 1e10))
-        ego = new Object at (0,0,0), with visibleDistance 1
-        foo = new Object in workspace, visible,
-            with shape SpheroidShape(dimensions=(0.2,0.2,0.2))
+        ego = new Object at (0,0,0), with visibleDistance 1, with allowCollisions True
+        foo = new Object in workspace,
+            with shape SpheroidShape(dimensions=(2,2,2)),
+            with allowCollisions True, visible
         param p = foo.position
         """
     )
     positions = [sampleParamP(scenario, maxIterations=100) for i in range(30)]
-    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 1.1 for pos in positions)
+    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 2 for pos in positions)
     assert any(pos.distanceTo(Vector(0, 0, 0)) >= 1 for pos in positions)
 
     # requireVisible with offset
     baseOffsetVal = 0.0001
     scenario = compileScenic(
         f"""
         workspace = Workspace(RectangularRegion(0@0, 0, 1e10, 1e10))
-        ego = new Object at (0,0,0), with visibleDistance 1
-        foo = new Object on workspace, with requireVisible True,
-            with shape SpheroidShape(dimensions=(0.2,0.2,0.2)),
+        ego = new Object at (0,0,0), with visibleDistance 1, with allowCollisions True
+        foo = new Object on workspace,
+            with shape SpheroidShape(dimensions=(2,2,2)),
+            with allowCollisions True,
+            with requireVisible True,
             with baseOffset (0,0,{baseOffsetVal}), with contactTolerance 0
         param p = foo.position
         """
     )
     positions = [sampleParamP(scenario, maxIterations=100) for i in range(30)]
-    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 1.1 for pos in positions)
+    assert all(pos.distanceTo(Vector(0, 0, 0)) <= 2 for pos in positions)
     assert any(pos.distanceTo(Vector(0, 0, 0)) >= 1 for pos in positions)
     assert all(pos.z == -baseOffsetVal for pos in positions)
 
     # visible with offset
-    baseOffsetVal = 0.0001
     scenario = compileScenic(
         f"""
         workspace = Workspace(RectangularRegion(0@0, 0, 1e10, 1e10))
-        ego = new Object at (0,0,0), with visibleDistance 1
-        foo = new Object on workspace, visible,
-            with shape SpheroidShape(dimensions=(0.2,0.2,0.2)),
+        ego = new Object at (0,0,0), with visibleDistance 1, with allowCollisions True
+        foo = new Object on workspace,
+            with shape SpheroidShape(dimensions=(2,2,2)),
+            with allowCollisions True, visible,
```

Comment:
The `visible` is hard to notice here: I'd move it back to being right after `on workspace`.